libcusp/include/galois/graphs/MiningPartitioner.h

```diff
@@ -1154,7 +1151,8 @@ class MiningGraph : public DistGraph<NodeTy, EdgeTy> {
     }
   }
 
-  virtual bool is_vertex_cut() const { return false; }
+private:
```

Comment:
Move this to be close to the other impl functions in the code so a dev doesn't have to jump around the file looking for them. It also makes it so you don't have to declare private again.

Reply:
I intentionally kept it where it was to make the changes clear for your review :p

Now that the change is acceptable, I'll make it compact.

Reply:
`is_vertex_cut` is originally using underscores so I would not change it to the camel case for now.

Reply:
That's fine